docs/source/cn/tensor.py

```diff
@@ -0,0 +1,36 @@
+import oneflow
+from docreset import reset_docstr
+
+reset_docstr(
+    oneflow.tensor,
+    r"""
+    用数据构造一个张量，如果设置了 :attr:`placement` 和 :attr:`sbp` ，则返回consistent tensor，
+        否则返回一个 local tensor 。
+       
+    参数：
+        - **data**: 张量的初始数据。可以是列表、元组、NumPy ndarray、标量或张量。
+
+    关键词参数：
+        - **dtype** (oneflow.dtype, 可选)：返回张量的所需数据类型。默认值：如果没有，则从数据推断数据类型。
+        - **device** (oneflow.device, 可选)：返回张量的所需设备。如果placement 和sbp 为None，则使用当前cpu。
+        - **placement** (oneflow.placement, 可选)：返回张量的理想位置。
```

Comment:
```suggestion
        - **placement** (oneflow.placement, 可选)：设置返回张量的 placement 属性。
```
原文：
> the desired placement of returned tensor.

如果觉得不好翻译的，可以和我或其它人讨论，但是不要随意翻译……（为什么 desired 被翻译成了”理想“）